electrum/i18n.py

```diff
@@ -26,6 +26,8 @@
 
 import gettext
 
+from PyQt5.QtCore import QLocale
```

Comment:
you must not assume that qt is available, in the core lib.

Reply:
You right. I also noticed this, after CI failed.

Reply:
Can you move the function to e.g. `electrum.gui.qt.util` in stead?